src/components/layouts/pages/inspect/Inspect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getPdfFiles } from '../../../../store/pdfFiles/selectors';
+import connect from 'react-redux/es/connect/connect';
+import { Document, Page, pdfjs } from 'react-pdf';
+
+import './Inspect.scss';
+
+pdfjs.GlobalWorkerOptions.workerSrc = '../pdf.worker.js';
+
+function Inspect(props) {
+    const { files } = props;
+
+    const [pageNumber] = useState(1);
+
+    const onLoadSuccess = page => {
+        page.getOperatorList().then(data => {
+            console.log('Bboxes: ', data.argsArray[data.argsArray.length - 1]);
```

Comment:
Could you please also log structure tree somewhere, just to make sure it is still available?